chaincode/src/test/java/de/upb/cs/uc4/chaincode/CourseChaincodeTest.java

```diff
@@ -2,6 +2,7 @@
 
 import de.upb.cs.uc4.chaincode.model.Course;
 import de.upb.cs.uc4.chaincode.model.Error;
+import de.upb.cs.uc4.chaincode.util.GsonWrapper;
```

Comment:
Was this an error before? Since you did not change anything else in this file, this is either a fix for a previous error, or an unused import.

Reply:
Not sure abut this, but courses chaincode is gone now anyway....

Reply:
Not sure, courses are gone now anyway...